recipes-bsp/uefi/l4t-launcher-extlinux.bb

```diff
@@ -20,10 +20,14 @@ def compute_dependencies(d):
     initramfs_image = d.getVar('INITRAMFS_IMAGE') or ''
     if initramfs_image != '' and (d.getVar('INITRAMFS_IMAGE_BUNDLE') or '') != '1':
         deps += " %s:do_image_complete" % initramfs_image
+    if d.getVar("PREFERRED_PROVIDER_virtual/dtb"):
+        deps += " virtual/dtb:do_deploy"
     return deps
 
 
 PATH =. "${STAGING_BINDIR_NATIVE}/tegra-flash:"
+DEPLOY_DIR_IMAGE_DTB = "${DEPLOY_DIR_IMAGE}"
+DEPLOY_DIR_IMAGE_DTB:append = "${@'/devicetree' if d.getVar('PREFERRED_PROVIDER_virtual/dtb') else ''}"
```

Comment:
Rather than using `:append` here, can you just put this directly into the setting above?  Otherwise, if someone uses a bbappend to modify `DEPLOY_DIR_IMAGE_DTB`, the append will be applied to their modified setting, which isn't really what's intended here.

Reply:
Yeah, this is awkward. I'll clean this up.